### PR TITLE
fix(documentation): shows example code for first canvas where it was missing

### PR DESCRIPTION
**Pull request by b1aserlu**

### CODE REVIEW

File: packages/documentation/src/stories/foundation/layout/grid/grid.docs.mdx [hunk 1]
```diff
@@ -20,7 +20,7 @@ import SampleScssMixins from './grid-scss-mixins.sample.scss?raw';
 
 Our grid system uses a series of containers, rows, and columns to layout and align content. It’s built with flexbox and is fully responsive. Below is an example and an in-depth explanation for how the grid system comes together.
 
-<Canvas of={GridStories.Basis} />
+<Canvas sourceState="shown" of={GridStories.Basis} />
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/foundation/layout/grid/grid.docs.mdx:23]: I would recommend expanding the source only on the first canvas on components, but leave foundation pages as they are.

File: packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx [hunk 2]
```diff
@@ -27,7 +27,7 @@ Bootstrap offers sizing classes with the suffixes bellow. They allow you to set
 
 #### Example
 
-<Canvas of={SizingStories.SizesPercent} />
+<Canvas sourceState="shown" of={SizingStories.SizesPercent} />
```
Comment (oliverschuerch) [inline packages/documentation/src/stories/utilities/sizing/sizing.docs.mdx:30]: I would recommend expanding the source only on the first canvas on components, but leave foundation pages as they are.

**Review by oliverschuerch — CHANGES_REQUESTED** [review]
set `sourceState="shown"` also for the following pages:
- internet-header breadcrumbs `docs` and `custom-items`
- internet-header footer `docs` and `custom-items`
- internet-header header `docs`, `custom-items`, `language switch overrides` and `os flyout overrides`

File: packages/documentation/src/stories/foundation/layout/grid/grid.docs.mdx (hunk 1 above)
Comment (b1aserlu) [reply packages/documentation/src/stories/foundation/layout/grid/grid.docs.mdx:23]: For the moment all the pages wil be have the source code shown. Maybe this wil change in the future

**Review by oliverschuerch — APPROVED** [review]

**Pull request merged**